pkgs/applications/misc/fuzzel/default.nix

```
@@ -22,16 +22,16 @@
 
 assert svgSupport -> enableCairo;
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "fuzzel";
-  version = "1.9.2";
+  version = "1.10.0";
 
   src = fetchFromGitea {
     domain = "codeberg.org";
     owner = "dnkl";
-    repo = pname;
-    rev = version;
-    hash = "sha256-X1P/ghX97KCQcrNk44Cy2IAGuZ8DDwHBWzh1AHLDvd4=";
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "fuzzel";
```

`fuzzel` is a fixed value here, and you'll not expect something like `overrideAttrs { pname = "fuzzel-wrapped"; }` to break.